privaterelay/settings.py

```diff
@@ -29,6 +32,9 @@
 
 import dj_database_url
 
+if TYPE_CHECKING:
+    import wsgiref.headers
```

Comment:
Might be my unfamiliarity, but it's not clear to me why this is needed - might warrant a comment.

Reply:
Oh hmm, I guess it's just to get at a type definition without including it at runtime, similar to TypeScript's [type-only imports](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export).

Reply:
Yes, it is a good analogy. From [the docs](https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING):

> A special constant that is assumed to be `True` by 3rd party static type checkers. It is `False` at runtime.

I thought `TYPE_CHECKING` would come up earlier, but this is the first time I needed it in our code.